### ISSUE TITLE
anthropometry

### ISSUE THREAD

**Issue by websocketing** [akshaya2689/body-measuring-system#1]
I'm glad to see this project. I wonder if this project supports anthropometry? I need to measure the human parts of the generated manikin. I hope I can get your reply and information.

**Comment by akshaya2689** [comment]
> I'm glad to see this project. I wonder if this project supports anthropometry? I need to measure the human parts of the generated manikin. I hope I can get your reply and information.

It is for measuring body . From this we get  measurements(cm) of legs, hands, shoulder and hip. I think it does not support all elements of Anthropometry but it supports measuring body circumferences  form given height. We can measure any human model with accurate height and  high photo quality .

**Comment by websocketing** [comment]
So can you measure waist circumference, hip circumference and height? I want to measure many parts.

**Comment by websocketing** [comment]
@akshaya2689

**Comment by akshaya2689** [comment]
> So can you measure waist circumference, hip circumference and height? I want to measure many parts.
No. From this we can't measure body circumference. only leg hand shoulder and hip is measured with respect to the key points.
